target/README.md

```diff
@@ -13,85 +13,113 @@
-| /        |             |
-| /        |             |
+## Getting started
 
-The following lifecycle actions are available:
+### Prerequisites
 
-| Action                                  | Description                               |
-| --------------------------------------- | ----------------------------------------- |
-| create.ps1                              | PowerShell _create_ lifecycle action      |
-| delete.ps1                              | PowerShell _delete_ lifecycle action      |
-| disable.ps1                             | PowerShell _disable_ lifecycle action     |
-| enable.ps1                              | PowerShell _enable_ lifecycle action      |
-| update.ps1                              | PowerShell _update_ lifecycle action      |
-| permissions/groups/grantPermission.ps1  | PowerShell _grant_ lifecycle action       |
-| permissions/groups/revokePermission.ps1 | PowerShell _revoke_ lifecycle action      |
-| permissions/groups/permissions.ps1      | PowerShell _permissions_ lifecycle action |
-| resources/groups/resources.ps1          | PowerShell _resources_ lifecycle action   |
-| configuration.json                      | Default _configuration.json_              |
-| fieldMapping.json                       | Default _fieldMapping.json_               |
+<!--
+Describe the specific requirements that must be met before using this connector, such as the need for an agent, a certificate or IP whitelisting.
 
-## Getting started
+**Please ensure to list the requirements using bullet points for clarity.**
 
-### Provisioning PowerShell V2 connector
+Example:
 
-#### Correlation configuration
+- **SSL Certificate**:<br>
+  A valid SSL certificate must be installed on the server to ensure secure communication. The certificate should be trusted by a recognized Certificate Authority (CA) and must not be self-signed.
+- **IP Whitelisting**:<br>
+  The IP addresses used by the connector must be whitelisted on the target system's firewall to allow access. Ensure that the firewall rules are configured to permit incoming and outgoing connections from these IPs.
+-->
 
-The correlation configuration is used to specify which properties will be used to match an existing account within _{connectorName}_ to a person in _HelloID_.
+### Connection settings
 
-To properly setup the correlation:
+The following settings are required to connect to the API.
 
-1. Open the `Correlation` tab.
+| Setting  | Description                        | Mandatory |
+| -------- | ---------------------------------- | --------- |
+| UserName | The UserName to connect to the API | Yes       |
+| Password | The Password to connect to the API | Yes       |
+| BaseUrl  | The URL to the API                 | Yes       |
+
+### Correlation configuration
 
-2. Specify the following configuration:
+The correlation configuration is used to specify which properties will be used to match an existing account within _{connectorName}_ to a person in _HelloID_.
```

Comment:
Ik zou in deze change ook gelijk de waarde in het Create.ps1 script aanpassen, mbt het Correlation Field:
PLus misschien nog een klein beetje uitleg hier in de readme. _(Al staat het al wel beschreven in de HID documenatie)_
```
        $correlationField = $actionContext.CorrelationConfiguration.PersonField
        $correlationValue = $actionContext.CorrelationConfiguration.accountFieldValue
```

Reply:
Die trek ik er liever even los van om het pr zuiver te houden.